package.json

```diff
@@ -32,6 +32,7 @@
     "cache-loader": "^4.1.0",
     "electron": "^10.3.0",
     "electron-builder": "^22.9.1",
+    "electron-devtools-installer": "^3.1.1",
```

Comment:
I already have `"electron-devtools-installer": "^3.1.0",` in `main` -- just curious why it doesn't appear here.

Reply:
Because it got merged in #77 — see https://github.com/mattfarina/rd/commit/56358ca1871bda62221bbf98d25220ea59995a69#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R51

I'll need to rebase to resolve merge conflicts anyway.